cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cooler.rename_chroms(clr, {chrom:f'chr{chrom}' for chrom in clr.chromnames}) #Add 'chr' from chromosome names to follow UCSC convention
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>